src/routes/(docs)/Table/ExampleTable.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+	import { Table, TableCell, TableRow, Typography } from '$lib';
+</script>
+
+<Table striped>
+	<Typography as="caption" variant="caption" class="mb-2">
+		Shopping list for 1 hour bagels
+	</Typography>
+	<TableRow>
+		<TableCell header>#</TableCell>
+		<TableCell header>Description</TableCell>
+		<TableCell header number>Price/kg</TableCell>
+		<TableCell header number>Quantity</TableCell>
+		<TableCell header number>Subtotal</TableCell>
+	</TableRow>
+	<TableRow>
+		<TableCell number>1</TableCell>
+		<TableCell>Flour</TableCell>
+		<TableCell number>£0.80</TableCell>
```

Comment:
Pedant ```&pound;``` and not £ please

Reply:
can do, not sure if it affects your opinion on this but svelte will actually automatically use the appropriate html entity where needed